templates/go/http/faas/main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	function "function"
+
+	"knative.dev/func/runtime"
+)
+
+func main() {
```

Comment:
We should add some comment here pointing users to handle.go. So they don't try to write the function code here.